### PR TITLE
placate rubocop 07/03

### PR DESCRIPTION
**Pull request by elguapogordo**
fixed 65 rubocop violations
0 rubocop violations
39 examples, 0 failures

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class UsersController < ApplicationController
-  def show
```
Comment (elguapogordo) [inline app/controllers/users_controller.rb:2]: empty space

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class UsersController < ApplicationController
-  def show    
-    if !current_user.token.nil?
```
Comment (elguapogordo) [inline app/controllers/users_controller.rb:3]: example of a boolean with double logic

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-      @repos = SearchResults.new.repos(current_user.token)
-      @followers = SearchResults.new.followers(current_user.token)
-      @followed_users = SearchResults.new.following(current_user.token)
-    end
+  def show
+    return unless current_user.token
```
Comment (elguapogordo) [inline app/controllers/users_controller.rb:3]: boolean simplified

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+
+    @repos = SearchResults.new.repos(current_user.token)
+    @followers = SearchResults.new.followers(current_user.token)
+    @followed_users = SearchResults.new.following(current_user.token)
```
Comment (elguapogordo) [inline app/controllers/users_controller.rb:7]: dropped the `if/end` block, added a return, and dropped the lines back to proper indentation

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (Joshua-Tukman) [reply app/controllers/users_controller.rb:2]: Easy fix

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (Joshua-Tukman) [inline app/controllers/users_controller.rb:3]: Much cleaner!

File: app/facades/tutorial_facade.rb [hunk 5]
```diff
@@ -17,7 +17,7 @@ def next_video
   end
 
   def play_next_video?
-    !(current_video.position >= maximum_video_position)
```
Comment (Joshua-Tukman) [inline app/facades/tutorial_facade.rb:20]: Thank you rubocop!

File: app/models/user.rb [hunk 6]
```diff
@@ -3,8 +3,8 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates_presence_of :password
-  validates_presence_of :first_name
+  validates :password, presence: true
+  validates :first_name, presence: true
```
Comment (Joshua-Tukman) [inline app/models/user.rb:7]: I personally like the old method better

File: app/poros/github_user.rb [hunk 7]
```diff
@@ -1,8 +1,8 @@
 class GithubUser
-    attr_reader :name, :url
+  attr_reader :name, :url
 
-    def initialize(info)
-        @name = info[:login]
```
Comment (Joshua-Tukman) [inline app/poros/github_user.rb:5]: I just updated my preferences in my text editor. Hopefully the extra tabs will stop

**Review by Joshua-Tukman — APPROVED** [review]
Great work. We're making good progress on this so far

**Pull request merged**